docs/application/native/guides/multimedia/raw-audio.md

```diff
@@ -28,8 +28,10 @@ Your application must define the following PCM data settings:
 - Audio sample type:
   - Unsigned 8-bit PCM
   - Signed 16-bit little endian PCM
+  - Signed 24-bit little endian PCM
+  - Signed 24-bit packed in 32-bit little endian PCM
```

Comment:
There are several ways to describe sample type: 
https://askubuntu.com/questions/421607/how-to-convert-an-audio-streams-format
http://0pointer.de/lennart/projects/pulseaudio/doxygen/sample.html

It would be better to write like http://0pointer.de/lennart/projects/pulseaudio/doxygen/sample.html, because a developer has to set the value with enum.

For instance, you can explain like below:
 
AUDIO_SAMPLE_TYPE_S16_LE:  Signed 16 integer bit PCM, little endian.

Reply:
updated done